cita-chain/src/main.rs

```diff
@@ -133,8 +134,8 @@ fn main() {
 
     let nosql_path = DataPath::nosql_path();
     trace!("nosql_path is {:?}", nosql_path);
-    let db_config = DatabaseConfig::with_columns(db::NUM_COLUMNS);
-    let db = Database::open(&db_config, &nosql_path).unwrap();
+    let db_config = DatabaseConfig::with_category_num(db::NUM_COLUMNS);
```

Comment:
`COLUMN`  is a proper noun from **Storage**, so I think the name `with_columns` is better than `with_category_num`.

Reply:
You can check the cita-database. All the interface use the category as the params.
category is a abstract of column. this column is the noun fo rocksdb.

We can discuss it in a issue of cita-database.

Reply:
ok.